node/src/service.rs

```diff
@@ -482,7 +482,7 @@ where
 		relay_chain_slot_duration,
 		proposer,
 		collator_service,
-		authoring_duration: Duration::from_millis(1500),
+		authoring_duration: Duration::from_millis(2000),
```

Comment:
I'm not sold on this change still. https://wiki.polkadot.network/docs/maintain-guides-async-backing#phase-2---update-parachain-nodes Says 1,500, with the possible for two seconds, but it's more risky.

Note, [polkadot-parachain](https://github.com/paritytech/polkadot-sdk/blob/c4b3c1c6c6e492c4196e06fbba824a58e8119a3b/cumulus/polkadot-parachain/src/service.rs#L741) and [hyberbridge](https://github.com/polytope-labs/hyperbridge/blob/d4fffd7e6672ca3bd1a927c9fc677e14c5066186/parachain/node/src/service.rs#L476) both use 1,500

Reply:
> Note: Set authoring_duration to whatever you want, taking your own hardware into account. But if the backer who should be slower than you due to reading from disk, times out at two seconds your candidates will be rejected.

https://wiki.polkadot.network/docs/maintain-guides-async-backing#phase-1---update-parachain-runtime
https://github.com/paritytech/polkadot-sdk/blob/c4b3c1c6c6e492c4196e06fbba824a58e8119a3b/templates/parachain/node/src/service.rs#L215
https://github.com/paritytech/polkadot-sdk/pull/4295 approved by Kian/Basti etc

Reply:
reverted